site/content/blog/incorporating-external-resources-in-kapp.md

```diff
@@ -0,0 +1,161 @@
+---
+title: "Incorporating external resources in kapp"
+slug: incorporating-external-resources-in-kapp
+date: 2021-12-24
+author: Praveen Rewar
+excerpt: "How to use kapp's exists annotation to wait for resources created by external agencies"
```

Comment:
may be `by external agencies` => `outside of direct configuration`?

Reply:
Makes sense, will update it.

Reply:
Done.